conans/client/downloaders/caching_file_downloader.py

```diff
@@ -64,8 +64,21 @@ def _try_download_from_backup(self, url, conanfile, h, cached_path, sha256, **kw
                 # TODO: What happens if sha256 missmatch?
                 pass
         if not found_in_backup:
-            # TODO: Policy to control this behaviour
-            pass
+            policy = conanfile.conf.get("tools.backup_sources:cache_miss_policy", check_type=str,
+                                        default="ignore")
+            # TODO: Think about what is the best default
+            message = f"Sources from {url} not found in remote backup sources server(s)"
+            if policy == "raise":
+                raise ConanException(message)
+            elif policy == "warn":
+                conanfile.output.warning(message)
+            elif policy == "ignore":
+                # Do nothing, we're fine with looking externally for the files
+                pass
+            else:
+                conanfile.output.warning("Backup sources cache missed but "
```

Comment:
Maybe better make it a hard error, for an unrecognized ``cache_miss_policy`` value?